addons/udes_stock/README.md

```diff
@@ -28,145 +28,160 @@
 | ---------------- | ------- | -------------------------------------------------------- |
-| id               | int     | |
-| barcode          | string  | |
-| name             | string  | |
-| quant_ids | [{stock.quants}]  | A list of all the quants at the given location. |
+| name              | int     | Disable translation instead of renaming |
+| active | [{stock.quants}]  | Add tracking or archiving |
 
 ### Quants (model: stock.quant)
 
 Physical instances of products at a location are modelled as quants. Short of "quantities of stock", these are used to record stock levels are various parts of the warehouse. Using the analogy of object-orientated programming, products = classes, quants = objects.
 
-| Field Name  | Type          | Description |
-| ----------- | ------------- | ----------- |
-| id | int | |
-| package_id | stock.quant.package | (see representation of the packages below) |
-| product_id | product.product | (see representation of the products above) |
-| location_id | stock.location | (See representation of the location above) |
-| quantity | float | The physical quantity of the stock |
-| reserved_quantity | float | The number of this quantity that has been reserved |
+Additional Details:
+- _description = 'Quant'
+
+| Helpers | Description |
+| ---------------- | ------- | 
+| _gather | |
+| get_quantity | Returns the total quantity of the quants in self |
+| get_quantities_by_key | Returns a dictionary with the total quantity per product, mapped by product_id |
+| create_picking | Create a picking from Quants |
 
 ### Packages (model: stock.quant.package)
 
 Physical packages of products. They can be used to represent parcels sent to customers, a pallet of products sent from a supplier, a tote used to pick products so they can be sent, or any combination of the above.
 
+Additional Details:
+- _description = "Package"
+- Additionally Inherits from: mixin.stock.model
 
-| Field Name       | Type    | Description                                              |
-| ---------------- | ------- | -------------------------------------------------------- |
-| id               | int     | |
-| name             | string  | |
-| package_id       | int     | ID of the parent package if it exists |
-| children_ids     | [{stock.quant.package}] | Children packages information if there is any |
-| quant_ids           | [{stock.quant}] | Stock inside the package |
+| Field Name   | Description | Value |
+| ------------ | ------ | ------ | 
+| MSM_CREATE   | Can create new packages | True |
 
+| Helpers | Description |
+| ------------ | ------ |
+| get_quantities_by_key | This function computes the product quantities for the given package grouped by a key |
+| mls_can_fulfil | Returns mls which the package can fulfil. If the product_qty of the mls is larger than in the package (i.e. in self) the mls will be split |
+| has_same_content | Compare the content of current package with the content of another package | 
```

Comment:
We have removed `ensure_one` so now it is packages vs packages instead of just package vs package.